resources/resolver.go

```diff
@@ -637,6 +637,41 @@ func ResolveResources(
 	return &foundResources, nil
 }
 
+func (rsrcs *Resources) ResolveSplitRegex() *string {
+	var splitRegex *string
+	if tokenizerData, ok := (*rsrcs)["tokenizer.json"]; ok {
+		var tokenizerMap map[string]interface{}
+		if tokenizerData.Data != nil {
+			if json.Unmarshal(*tokenizerData.Data, &tokenizerMap) != nil {
+				log.Fatal("Error unmarshalling tokenizer.json")
+			}
+		}
+		if preTok, ok := tokenizerMap["pre_tokenizer"]; ok && preTok != nil {
+			preTokMap := preTok.(map[string]interface{})
+			if pretokenizers, ok := preTokMap["pretokenizers"]; ok &&
+				pretokenizers != nil {
+				pretokenizersList := pretokenizers.([]interface{})
+				for _, v := range pretokenizersList {
+					vIntf := v.(map[string]interface{})
+					if vIntf["type"] == "Split" {
+						pattern := vIntf["pattern"].(map[string]interface{})
+						if pattern["Regex"] != nil {
+							splitRegexVal := pattern["Regex"].(string)
+							// Fix lookbacks
```

Comment:
<migrated offline clarification discussion>
me: The "fix" here is a performance improvement?
wes: No. It's a negative lookback. And golang's regex doesn't support it.